core/modules/image/tests/image.test

```diff
@@ -170,6 +170,22 @@ class ImageStylesPathAndUrlUnitTest extends BackdropWebTestCase {
   }
 
   /**
+   * Test that we do not pass an array to backdrop_add_http_header.
+   */
+  function testImageContentTypeHeaders() {
+    $files = $this->backdropGetTestFiles('image');
+    $file = array_shift($files);
+    // Copy the test file to private folder.
+    $private_file = file_copy($file, 'private://', FILE_EXISTS_RENAME);
```

Comment:
file_copy() in Backdrop requires a File entity. As this is an unmanaged file, this needs to be:

```
$private_uri = file_unmanaged_copy($file->uri, 'private://test.png');
```